Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperations/TestOperationResult.swift

```diff
@@ -184,66 +184,71 @@ struct ErrorResult: Equatable, Decodable {
         self.errorLabelsOmit = try container.decodeIfPresent([String].self, forKey: .errorLabelsOmit)
     }
 
-    public func checkErrorResult(_ error: Error) throws {
```

Comment:
To help debugging, I added a description to all of the assertions in here. I also removed the `throws` from everything because nothing in here actually throws an error.